VirtoCommerce.PricingModule.Web/Scripts/blades/prices-list.tpl.html

```diff
@@ -39,6 +41,13 @@
         </div>
     </div>
 </script>
+<script type="text/ng-template" id="ui-grid/dateCellTemplate">
```

Comment:
I think using a date-time picker control to enter dates would make better the  UX  of that solution.